relay-general/src/protocol/debugmeta.rs

```diff
@@ -559,6 +583,29 @@ mod tests {
         assert_eq!(json, image.to_json_pretty().unwrap());
     }
 
+    #[test]
+    fn test_debug_image_jvm_based_roundtrip() {
+        let json = r#"{
+  "debug_id": "395835f4-03e0-4436-80d3-136f0749a893",
+  "other": "value",
+  "type": "jvm"
+}"#;
+        let image = Annotated::new(DebugImage::Jvm(Box::new(JvmDebugImage {
+            debug_id: Annotated::new("395835f4-03e0-4436-80d3-136f0749a893".parse().unwrap()),
+            other: {
+                let mut map = Object::new();
+                map.insert(
+                    "other".to_string(),
+                    Annotated::new(Value::String("value".to_string())),
+                );
+                map
```

Comment:
Since it's a map, instead of `Object::new`, let's use a map type. You can also ease the map creation and make it more readable with `::from`, see [example](https://github.com/getsentry/relay/blob/309fac640c0597d83c15ac60f9f7607d88de891b/relay-metrics/src/aggregation.rs#L2422).

Reply:
Should I go ahead an change all of the tests in the file to use `Map::new()` instead of `Object::new()`?

Reply:
Nah, that's out of scope for this PR, let's keep it focused 😄.